eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/eventlist/view/EventListMenu.java

```diff
@@ -60,6 +62,8 @@ public boolean onOptionsItemSelected(MenuItem item)
         }
         else if (item.getItemId() == R.id.schedjoules_event_list_menu_feedback)
         {
+            // TODO Needed to close the menu explicitly, otherwise the popup was there when came back from Feedback. Why is this, remove if possible.
```

Comment:
check out how it's done in the event details MF. IIRC you need to "post" the handler so it's enqueued in the message queue.